app/code/Magento/Catalog/Model/Product/Attribute/Repository.php

```diff
@@ -1,319 +0,0 @@
-<?php
```

Comment:
We can't remove attribute repository according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)